src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -110,4 +117,22 @@ public function push($stat, $value, $time = null)
             $time ?: time()
         );
     }
+
+    /**
+     * @internal
+     * @todo Makes this method private, move pushStat() logic into this one afterward
+     *
+     * @param $stat
+     * @param $value
+     * @param int|null $time
+     */
+    public function push($stat, $value, $time = null)
+    {
+        trigger_error(
+            'This method is used for internal usage only. It will be removed from public API on the next major version',
```

Comment:
this method is not for internal use (it is not used internally anymore). It is deprecated in your change